### ISSUE TITLE
Set Up Initial Documentation

### ISSUE THREAD

**Issue by erikaheidi** [do-community/creatorlinks#4]
Set up documentation. Priorities:

- About
- Contributing.md file
- **How to set up the development environment**
- How to deploy to production on App Platform and other methods (less priority initially)

**Comment by lynmuldrow** [comment]
I can tackle this one, too. Happy to gain initial thoughts/ideas around communicating this content to mock something up for review.

**Comment by erikaheidi** [comment]
Hey @lynmuldrow thanks for grabbing this one! 

I think that for the end user we must have something showing details of how to set up / deploy within a real web server, but maybe that can wait a bit until we have more constrains about the setup. In terms of practicality, I was considering using readthedocs.org to host the docs, I used it in the past and it sounds like something good for the long term. 

By the way, I'm totally open to any suggestions and ideas.

**Comment by lynmuldrow** [comment]
I totally agree and think that it would be wise to wait until we have a flow that makes sense for end users. 

I think readthedocs is a great choice and is community-minded, which we'd like to def promote.
